objectivec/GPBUtilities.m

```diff
@@ -49,6 +49,19 @@ static void AppendTextFormatForMessage(GPBMessage *message,
                                        NSMutableString *toStr,
                                        NSString *lineIndent);
 
+// Are two datatypes the same basic type representation (ex Int32 and SInt32).
+// Marked unused because currently only called from asserts/debug.
+static BOOL GPBDataTypesEquivalent(GPBDataType type1,
```

Comment:
Don't really need GPB on these since they are file private and not meant to ever be exposed.

Reply:
Fixed